### DISCUSSION TITLE
Not pre-installed extensions

### DISCUSSION THREAD

**Discussion by eawmaas** [xenocrat/chyrp-lite#196] [Q&A] [answered]
Is there a way to get a list of all not pre-installed extentions, mods, leafs etc. (let's say: functionalities) that you can add if you want?

**Comment by xenocrat** [comment, marked as answer]
Hello there,

Here are the published extensions I know of:
1. [reCAPTCHA v2](http://thethinkingman.com.au/2020/10/07/recaptcha-v2-module-for-chyrp-lite/)
2. [Content Security Policy](https://github.com/xenocrat/chyrp-csp)
3. [Alphabetize](https://github.com/xenocrat/chyrp-alphabetize)
4. [Mail To File](https://github.com/xenocrat/chyrp-mail_to_file)

I'm aware of quite a few custom themes created by Chyrp Lite users, but the authors have not published them.

**Reply by eawmaas** [reply]
Thank you!
